### PR TITLE
expose venue_type in search results, add venue_types filter

### PR DESCRIPTION
**Pull request by jbraswell**

### CODE REVIEW

**Comment by pjaudiomv** [comment]
idk if you plan on doing in another pass, but your gonna want to add the venue type to the translations here https://github.com/bmlt-enabled/bmlt-root-server/blob/master/main_server/local_server/server_admin/lang/en/server_admin_strings.inc.php#L418

**Comment by jbraswell** [comment]
> idk if you plan on doing in another pass, but your gonna want to add the venue type to the translations here https://github.com/bmlt-enabled/bmlt-root-server/blob/master/main_server/local_server/server_admin/lang/en/server_admin_strings.inc.php#L418

Are you sure? I didn't expose anything new in the admin ui. Everything is exactly the same. There is just a new field that comes back when you search for meetings, and a new column in the database.

**Comment by pjaudiomv** [comment]
> > idk if you plan on doing in another pass, but your gonna want to add the venue type to the translations here https://github.com/bmlt-enabled/bmlt-root-server/blob/master/main_server/local_server/server_admin/lang/en/server_admin_strings.inc.php#L418
> 
> Are you sure? I didn't expose anything new in the admin ui. Everything is exactly the same. There is just a new field that comes back when you search for meetings, and a new column in the database.

ok so you havent added the part where you can search for it through semantic interface

**Comment by jbraswell** [comment]
> > > idk if you plan on doing in another pass, but your gonna want to add the venue type to the translations here https://github.com/bmlt-enabled/bmlt-root-server/blob/master/main_server/local_server/server_admin/lang/en/server_admin_strings.inc.php#L418
> > 
> > 
> > Are you sure? I didn't expose anything new in the admin ui. Everything is exactly the same. There is just a new field that comes back when you search for meetings, and a new column in the database.
> 
> ok so you havent added the part where you can search for it through semantic interface

That's right. None of the edited files belong to the semantic interface.

File: main_server/local_server/server_admin/lang/en/server_admin_strings.inc.php [hunk 1]
```diff
@@ -421,6 +421,8 @@
         'shared_group_id_bigint' => 'Unused',
         'service_body_bigint' => 'Service Body ID',
         'weekday_tinyint' => 'Weekday',
+        'venue_type' => 'Venue Type',
```
Comment (pjaudiomv) [inline main_server/local_server/server_admin/lang/en/server_admin_strings.inc.php:424]: I have double vision

**Comment by jbraswell** [comment]
Can try this out at https://bmlt.sezf.org/main_server/semantic/

**Comment by jbraswell** [comment]
Added these fields to tomato here: https://github.com/bmlt-enabled/tomato/pull/121

**Review by alanb2718 — APPROVED** [review]
Looks good!  I tested this on the SEZF site, and also on a local copy of the WSZF data. Everything checked out, including testing what happens with meetings with inconsistent formats (just TC but not VM) - that is handled correctly. My only minor suggestion is in CONTRIBUTING.md to change 
`If you do want to access mysql, open another shell using`
to
`If you do want to access mysql, run the command`
(since this isn't opening another shell to run the command line interface to mysql, but just running it directly).

**Comment by jbraswell** [comment]
Sweet, I just fixed the sentence in `CONTRIBUTING.md`

**Pull request merged**